OONIProbeUITests/OONIProbeUITests.swift

```diff
@@ -7,58 +7,16 @@ class OONIProbeUITests: XCTestCase {
-            
-            let appDirs = try! FileManager.default.contentsOfDirectory(atPath: containerDir.path)
-
-            // ATTENTION BIG CAVEAT!
-            // Currently for the logic below to work you need to have installed (and run) OONI Probe at least once inside of the simulator for the target device. Once that is done it will work OK, but it won't work until then.
-            // When in doubt try rerunning the tests and they should work.
-            // XXX maybe a better way to do this is:
-            // https://developer.apple.com/documentation/foundation/filemanager/1412643-containerurl
-            var targetAppDir = containerDir
-            for appDir in appDirs {
-                targetAppDir = containerDir.appendingPathComponent(appDir)
-                if FileManager.default.fileExists(atPath: targetAppDir.appendingPathComponent("Documents/OONIProbe.db").path) {
-                    print("Success!!")
-                    print(targetAppDir)
-                    break
-                }
-            }
-            XCTAssertTrue(targetAppDir != containerDir, "Could not find targetAppDir")
-
-            let prefPath = targetAppDir.appendingPathComponent("Library/Preferences/org.openobservatory.ooniprobe.plist")
-            try? FileManager.default.removeItem(at: prefPath)
-            try! FileManager.default.copyItem(
-                atPath: fixturesDir.appendingPathComponent("org.openobservatory.ooniprobe.plist").path,
-                toPath: prefPath.path
-            )
-            let dbPath = targetAppDir.appendingPathComponent("Documents/OONIProbe.db")
-            print("Deleting \(dbPath.path)")
-            try? FileManager.default.removeItem(at: dbPath)
-            try? FileManager.default.removeItem(at: targetAppDir.appendingPathComponent("Documents/OONIProbe.db-wal"))
-            try? FileManager.default.removeItem(at: targetAppDir.appendingPathComponent("Documents/OONIProbe.db-shm"))
-
-            try! FileManager.default.copyItem(
-                atPath: fixturesDir.appendingPathComponent("OONIProbe.db").path,
-                toPath: dbPath.path
-            )
-            print("repo Root \(repoRoot)\n")
-            print("targetAppDir \(targetAppDir.path)\n")
-            print("sharedFolderURL \(containerDir.path)\n")
-            print("appID: \(appID)\n")
-        }
-        
         // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
         let app = XCUIApplication()
         setupSnapshot(app)
+
+        // Solution taken rom https://blog.codecentric.de/en/2018/06/resetting-ios-application-state-userdefaults-ui-tests/
+        //Preventing Informed Consent to appear
+        app.launchArguments += ["-first_run", "ok"]
+        app.launchArguments += ["-upload_result_manually_popup", "ok"]
+        app.launchArguments += ["UI-Testing"]
```

Comment:
Maybe for consistency you want to use either all dash case for the CLI argument or all snake case.

I personally prefer `dash-case` for CLI arguments, but I think it's the same.

Reply:
upload_result_manually_popup and first_run can't be changed as they are internal settings already in use

Reply:
Ok I see. Then maybe we should also use `snake_case` for `UI-Testing`?

Also maybe since it's a flag you want it to be `-enable_ui_testing`?

Reply:
Ok can be done. I used UI-Testing because it's what I found on the codecentric blog, but can be anything we want. There is to point out that `first_run` is an userdefaults of the app while  `UI-Testing` is a launch parameter

Reply:
Ok I see. Then maybe it's OK to use a different case. I would still suggest prefixing it with `-` to make it clear that it's a flag and maybe call it like: `-enable-ui-testing`

Reply:
The `-` prefix will inject it as UserDefault so it's not this case.
I will rename it to `enable-ui-testing` if you prefer, no big deal

Reply:
Yes, read the comment above of 12 days ago with the blog linked

Reply:
Will it do that even if the setting does not exist and if you are not passing the argument `ok`?

Reply:
it stores "ok" so won't show the onboarding